src/site/layouts/press_release.drupal.liquid

```diff
@@ -238,7 +238,7 @@
                         {{ fieldOffice.entity.fieldPressReleaseBlurb.processed }}
                     </section>
                     {% assign index = entityUrl.breadcrumb.length | minus: 2 %}
-                    <a onClick="recordEvent({ event: 'nav-secondary-button-click' });" href="{{ entityUrl.breadcrumb | getValueFromArrayObjPath: index, 'url.path' }}">See all news releases</a>
+                    <a onClick="recordEvent({ event: 'nav-secondary-button-click' });" href="{{ entityUrl.breadcrumb | getValueFromArrayObjPath: 2, 'url.path' }}">See all news releases</a>
```

Comment:
This works, but it assumes that the breadcrumb element we want - the last one - is always at index 2. What if the breadcrumb length changes?

Wouldn't it be safer to change line 240 to 
```
{% assign index = entityUrl.breadcrumb.length | minus: 2 %}
```
??

Also, please note that there's a block of code that only runs in production and contains the same bug, starting on line 102. We really should go through the code and remove any prod-specific code that we can.

Reply:
> This works, but it assumes that the breadcrumb element we want - the last one - is always at index 2. What if the breadcrumb length changes?
> 
> Wouldn't it be safer to change line 240 instead to
> 
> ??

Nice catch and yes this is better!

> 
> Also, please note that there's a block of code that only runs in production and contains the same bug, starting on line 102. We really should go through the code and remove any prod-specific code that we can.

Yes let's check with the team and fix these but in another issue